apis/azure/network/basic/composition.yaml

```diff
@@ -11,97 +11,91 @@ spec:
-        kind: VirtualNetwork
-        spec:
-          forProvider:
-            resourceGroupNameSelector:
-              matchControllerRef: true
-            addressSpace:
-              - 192.168.0.0/16
-      patches:
-        - type: PatchSet
-          patchSetName: providerConfigRef
-        - type: PatchSet
-          patchSetName: deletionPolicy
-        - type: PatchSet
-          patchSetName: network-id
-        - type: PatchSet
-          patchSetName: region
-        - fromFieldPath: spec.parameters.id
-          toFieldPath: metadata.name
-          transforms:
-            - type: string
-              string:
-                fmt: "%s-vnet"
-    - name: subnet
-      base:
-        apiVersion: network.azure.upbound.io/v1beta1
-        kind: Subnet
-        spec:
-          forProvider:
-            resourceGroupNameSelector:
-              matchControllerRef: true
-            virtualNetworkNameSelector:
-              matchControllerRef: true
-            addressPrefixes:
-              - 192.168.1.0/24
-            serviceEndpoints:
-              - Microsoft.Sql
-      patches:
-        - type: PatchSet
-          patchSetName: providerConfigRef
-        - type: PatchSet
-          patchSetName: deletionPolicy
-        - type: PatchSet
-          patchSetName: network-id
-        - type: PatchSet
-          patchSetName: region
-        - fromFieldPath: spec.parameters.id
-          toFieldPath: metadata.name
-          transforms:
-            - type: string
-              string:
-                fmt: "%s-sn"
+  - name: resource-group
+    base:
+      apiVersion: azure.upbound.io/v1beta1
+      kind: ResourceGroup
+    patches:
+    - type: PatchSet
+      patchSetName: providerConfigRef
+    - type: PatchSet
+      patchSetName: deletionPolicy
+    - type: PatchSet
+      patchSetName: network-id
+    - type: PatchSet
+      patchSetName: region
+  - name: virtual-network
+    base:
+      apiVersion: network.azure.upbound.io/v1beta1
+      kind: VirtualNetwork
+      spec:
+        forProvider:
+          resourceGroupNameSelector:
+            matchControllerRef: true
+          addressSpace:
+          - 192.168.0.0/16
+    patches:
+    - type: PatchSet
+      patchSetName: providerConfigRef
+    - type: PatchSet
+      patchSetName: deletionPolicy
+    - type: PatchSet
+      patchSetName: network-id
+    - type: PatchSet
+      patchSetName: region
+    - fromFieldPath: spec.parameters.id
+      toFieldPath: metadata.name
+      transforms:
+      - type: string
+        string:
+          fmt: "%s-vnet"
+  - name: subnet
+    base:
+      apiVersion: network.azure.upbound.io/v1beta1
+      kind: Subnet
+      spec:
+        forProvider:
+          resourceGroupNameSelector:
+            matchControllerRef: true
+          virtualNetworkNameSelector:
+            matchControllerRef: true
+          addressPrefixes:
+          - 192.168.1.0/24
+          serviceEndpoints:
+          - Microsoft.Sql
```

Comment:
is it leftover from platform-ref where we are making subnet ready for the database? do we want to keep it around?

Reply:
fixed